vllm/worker/hpu_model_runner.py

```diff
@@ -586,6 +607,8 @@ def __init__(
         self.bucketing_global_state = HPUBucketingGlobalState()
         self._setup_buckets()
         self._set_gc_threshold()
+        self.use_contiguous_pa = os.environ.get(
+            'VLLM_CONTIGUOUS_PA', 'true').lower() in ['1', 'true']
```

Comment:
We should not directly access the environment variable in this way. It makes environment variables hard to manage. Please add this variable to `vllm.envs` and use `vllm.envs.VLLM_XXX` here.

In addition, the name `VLLM_CONTIGUOUS_PA` is not informative enough. For example I have no idea what "PA" is. If this is a common term for Gaudi users, we could probably use the name `VLLM_USE_HPU_CONTIGUOUS_PA`.